modules/debug.js

```diff
@@ -1,31 +1,46 @@
+var KeyCode = {
+  DEBUG : 68, // 'D'
+  PAUSE : 80  // 'P'
+}
 var Debug = function()
 {
-  this.debugMode = true;
+  this.population = null;
+  this.debugMode = false;
+  this.paused = false;
   this.statsInterval = null;
 
   var self = this;
 
-  $("html").click(function()
+  $(document).bind("keyup", function(e)
   {
-    self.debugMode = !self.debugMode;
-
-    if(self.debugMode)
+    if(e.shiftKey && e.keyCode == KeyCode.DEBUG)
     {
-      self.Start();
+      self.debugMode = !self.debugMode;
+
+      (self.debugMode) ? self.Start() : self.Stop();
     }
-    else
+
+    if(e.shiftKey && e.keyCode == KeyCode.PAUSE)
     {
-      self.Stop();
+      self.paused = !self.paused;
+
+      (self.paused) ? self.Paused() : self.Resume();
     }
   });
 };
 
 // Might be necessary if we want to build more functionality to Debug class
 Debug.prototype = {
-  Start: function(population)
+  Init: function(population)
   {
-    Stats.show(population);
-    this.statsInterval = setInterval(function(){ Stats.show(population); }, 1000);
+    this.population = population;
+  },
+
+  Start: function()
+  {
+    var self = this;
+    Stats.show(self.population);
+    this.statsInterval = setInterval(function(){ Stats.show(self.population); }, 1000);
```

Comment:
we should also redraw population's angle, because currently
if i shift+P then shift+D, i dont see the angles